draft/spec/validation-codes.md

```diff
@@ -84,7 +84,7 @@
 | E073 | 'Empty directories MUST NOT appear under a storage root.' | https://ocfl.io/draft/spec/#E073
 | E074 | 'Although implementations may require multiple OCFL Storage Roots - that is, several logical or physical volumes, or multiple "buckets" in an object store - each OCFL Storage Root MUST be independent.' | https://ocfl.io/draft/spec/#E074
 | E075 | 'The OCFL version declaration MUST be formatted according to the NAMASTE specification.' | https://ocfl.io/draft/spec/#E075
-| E076 | '[The OCFL version declaration] MUST be a file in the base directory of the OCFL Storage Root giving the OCFL version in the filename.' | https://ocfl.io/draft/spec/#E076
+| E076 | 'There must be exaclty one version declaration file in the base directory of the OCFL Storage Root giving the OCFL version in the filename.' | https://ocfl.io/draft/spec/#E076
```

Comment:
typo: `exaclty`

Reply:
Thanks! Fixed in [5992508](https://github.com/OCFL/spec/pull/585/commits/5992508e8b81f3942d2374de253a9cfe7353c097)